examples/autobahn_client.nim

```diff
@@ -80,7 +80,7 @@ proc main() {.async.} =
 
       while ws.readystate != ReadyState.Closed:
         # echo back
-        let data = await ws.recv()
+        let data = await ws.recvMsg()
```

Comment:
I've run the autobahn test locally because the last report was not generated due to some issue in the CI.
the report shows many UTF-8 test is failing. the reason is autobahn test expect us to treat incoming message either as text or binary. but because we are using `recvMsg`, invalid received text message is not verified.